include/exadg/postprocessor/pointwise_output_generator_base.h

```diff
@@ -22,6 +22,8 @@
 #ifndef INCLUDE_COMPRESSIBLE_NAVIER_STOKES_POSTPROCESSOR_POINTWISE_OUTPUT_GENERATOR_BASE_H_
 #define INCLUDE_COMPRESSIBLE_NAVIER_STOKES_POSTPROCESSOR_POINTWISE_OUTPUT_GENERATOR_BASE_H_
 
+#include <exadg/utilities/extract_component_from_tensors.h>
```

Comment:
Still, I am not sure whether we should break here with the current ExaDG convention. I am open to change this in the future if other solutions are more intelligent, but for now I would like to keep the convention we currently have.

Reply:
I will change all includes to the exadg convention :)

Reply:
[376c941](https://github.com/exadg/exadg/pull/255/commits/376c941dacec7e145658a1b4dabfd7d77604f9a0)